commands/reset_password.ts

```diff
@@ -0,0 +1,27 @@
+import { inject } from "@adonisjs/core";
+import { BaseCommand, args } from "@adonisjs/core/ace";
+import type { CommandOptions } from "@adonisjs/core/types/ace";
+
+import ResetPasswordService from "#services/reset_password_service";
+
+export default class ResetPassword extends BaseCommand {
+  static commandName = "reset:password";
+  static description =
+    "Tries to reset the password for user associated with given email address";
+
+  static options: CommandOptions = {
+    startApp: true,
+  };
+
+  @args.string({
+    argumentName: "email",
+    description:
+      "Email address associated with user for whom the resetting is done",
```

Comment:
![this description is fire](https://media1.tenor.com/m/dxPl_UoR8J0AAAAC/fire-writing.gif)